packages/client/wallets/aa/src/types/Config.ts

```diff
@@ -58,50 +46,28 @@ export type ViemAccount = {
 
 type Signer = EIP1193Provider | Web3AuthSigner | ViemAccount;
 
-export interface WalletConfig {
+export interface EOAWalletConfig {
     signer: Signer;
 }
 
-export interface PasskeyCipher {
-    chain: BlockchainIncludingTestnet;
-    walletAddress: string;
-    cipher: Cipher;
+export function isEOAWalletConfig(config: WalletConfig): config is EOAWalletConfig {
+    return "signer" in config;
 }
 
-export interface LitProtocolCipherData {
-    pkpPublicKey?: string;
-    pkpEthAddress?: string;
-    cipherText?: string;
-    dataToEncryptHash?: string;
+export interface PasskeyWalletConfig {
+    name?: string;
 }
 
-export type EncryptInput = {
-    messageToEncrypt: string;
-    pkpPublicKey: string;
-    pkpEthAddress: string;
-    capacityDelegationAuthSig: AuthSig;
-};
-
-export type DecryptInput = {
-    pkpPublicKey: string;
-    pkpEthAddress: string;
-    cipherText: string;
-    dataToEncryptHash: string;
-    capacityDelegationAuthSig: AuthSig;
-};
-
-type Cipher = { method: "lit_protocol"; data: LitProtocolCipherData };
-
-export type BackwardsCompatibleChains = "goerli";
```

Comment:
Do we really need this? The generic it forced us to have everywhere was noisy and made the little TS preview of our classes mostly unusable.